signalfx-org-metrics/metrics.yaml

```diff
@@ -1,510 +1,512 @@
 sf.org.apm.numSpansDroppedThrottle:
-  brief: The number of spans SignalFx accepts beyond the allowed ingest volume.
-  description: 'The number of spans SignalFx accepts beyond the allowed ingest volume.
-    SignalFx drops spans it receives after the ingestion volume limit is reached.
-
-    Dimension(s): `orgId`
+  brief: The number of spans Splunk APM accepts beyond the allowed ingest volume.
+  description: |
```

Comment:
Why `|`?

Reply:
This is YAML notation. See https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines